SPECS/gpumon.spec

```diff
@@ -28,6 +27,7 @@ metrics.
 %autosetup -p1
 
 %build
+DESTDIR=%{buildroot} %{__make} mock
```

Comment:
I'd probably add a comment here too to explain that line that could appear surprising to people reading the spec file